kod/object/item/passitem/numbitem/food/cookie.kod

```diff
@@ -0,0 +1,53 @@
+% Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+% All rights reserved.
+%
+% This software is distributed under a license that is described in
+% the LICENSE file that accompanies it.
+%
+% Meridian is a registered trademark.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+Cookie is Food
+
+constants:
+
+   include blakston.khd
+
+resources:
+
+   include Cookie.lkod
+
+   cookie_name_rsc = "cookie"
+   cookie_icon_rsc = cookie.bgf
+   cookie_desc_rsc = \
+      "Baked by Daenks, this cookie is both warm and soft"
+      " to the touch. Your mouth waters at just the sight of it."
```

Comment:
"Your mouth waters at just the sight of it." should probably be "Your mouth waters at the very sight of it."